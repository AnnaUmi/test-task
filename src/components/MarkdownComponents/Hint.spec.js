import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Hint from './Hint.styles';

describe('Hint', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Hint />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
