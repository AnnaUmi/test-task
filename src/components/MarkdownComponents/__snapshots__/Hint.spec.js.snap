// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hint should match snapshot 1`] = `
.c0 {
  background-color: #fffacd;
  border: 1px solid #ffe4b5;
  padding: 10px 20px;
  margin: 30px 0;
}

<div
  className="c0"
/>
`;
